packages/ramda-extension/src/unfoldObjectDots.js

```diff
@@ -10,10 +9,14 @@ import {
 	call,
 	identity,
 	compose,
+	reduce,
+	mergeDeepRight,
 } from 'ramda';
 import applyCompose from './applyCompose';
 import splitByDot from './splitByDot';
 
+const mergeAllDeep = reduce(mergeDeepRight, {});
```

Comment:
https://ramda-extension.firebaseapp.com/docs/#mergeDeepRightAll

Reply:
Thanks, I didn't notice about this function